### PR TITLE
Scala 2.12 and Play 2.6

### PR DESCRIPTION
**Pull request by JamesMcIntosh**
Upgraded dependences to the latest and greatest, took a couple of changes for the crossbuild from 
https://github.com/peel/play-soap

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @JM-Agrimap,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by JamesMcIntosh** [comment]
When running `sbt test` locally the build succeeds, I'm not sure what the problems with the CI build is, some feedback would be good if I need to make other changes. Thanks!

**Comment by JamesMcIntosh** [comment]
signed CLA

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,11 +1,11 @@
 /*
  * Copyright (C) 2015-2017 Lightbend Inc. <https://www.lightbend.com>
  */
-addSbtPlugin("com.typesafe.play" % "interplay" % sys.props.get("interplay.version").getOrElse("1.1.2"))
+addSbtPlugin("com.typesafe.play" % "interplay" % sys.props.get("interplay.version").getOrElse("2.0.1"))
```
Comment (marcospereira) [inline project/plugins.sbt]: Use interplay 1.3.15 since 2.0.x adds Scala 2.13 which is not supported by Play 2.6.

File: build.sbt [hunk 2]
```diff
@@ -30,8 +30,13 @@ lazy val plugin = project
     libraryDependencies ++= Common.pluginDeps,
     addSbtPlugin("com.typesafe.play" % "sbt-plugin" % Common.PlayVersion),
     resourceGenerators in Compile += generateVersionFile.taskValue,
+    crossScalaVersions := Seq("2.11.11","2.12.5"),
```
Comment (marcospereira) [inline build.sbt]: Scala 2.12.6 is out! :-)

**Comment by lightbend-cla-validator** [comment]
Hi @JM-Agrimap,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by lightbend-cla-validator** [comment]
Hi @JM-Agrimap,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by JamesMcIntosh** [comment]
@marcospereira I've made your requested changes, now 2 of the 3 builds pass. The third fails with:

> Cannot add dependency 'org.scala-sbt#compiler-interface;1.1.4' to configuration 'component' of module org.scala-sbt.temp#temp-module-750703454be73072ce64f1c215ff85550284ae5b;1.1.4 because this configuration doesn't exist!

I've also made another change to remove most of the deprecation warnings around JavaConversions.
I have left the Logging Interceptor warnings as they are only deprecated because of a new runtime logging module in CFX, see:
* http://cxf.apache.org/docs/message-logging.html
* https://www.codenotfound.com/apache-cxf-logging-soap-request-response-fault-messages-example.html

**Comment by ihostage** [comment]
@marcospereira do you known how to fix this problem? We are waiting for `play-soap-client_2.12` 😄

**Comment by lightbend-cla-validator** [comment]
Hi @JM-Agrimap,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by marcospereira** [comment]
Hi @JamesMcIntosh and @ihostage,

I've submitted some new commits here and the problem is fixed. @JamesMcIntosh, hope you don't mind it. I did because this PR is open for a long time and I don't want to take more of your time to fix the problems here, so I did it myself. Anyway, build is green now:

https://travis-ci.org/playframework/play-soap/builds/397598321

Best.

**Comment by JamesMcIntosh** [comment]
Thanks very much Marcos for following up and fixing that last issue!

**Pull request merged**